package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "@adobe/alloy",
   "version": "2.2.0",
-  "description": "Client-Side SDK for Unified Data Collection",
-  "main": "src/core/index.js",
+  "description": "Adobe Experience Platform Web SDK",
+  "module": "dist/index.js",
+  "files": [
+    "dist/index.js"
+  ],
```

Comment:
This array limits the files that are uploaded as part of the NPM library. "package.json", "LICENSE", and "README.md" are included automatically.